documentation/IDTA-01002-3/modules/ROOT/pages/specification/interfaces.adoc

```diff
@@ -1920,7 +1926,29 @@ It is the predefined name “_globalAssetId_” that would refer to the _AssetIn
 |yes |link:https://admin-shell-io.github.io/aas-specs-antora/IDTA-01001/v3.1/spec-metamodel/core.html#SpecificAssetId[SpecificAssetId] |1..*
 5+h|Output Parameter
 |statusCode |Status code |yes |xref:specification/interfaces-payload.adoc#StatusCode[StatusCode] |1
-|payload |Asset identifier created successfully |yes |link:https://admin-shell-io.github.io/aas-specs-antora/IDTA-01001/v3.1/spec-metamodel/core.html#SpecificAssetId[SpecificAssetId] |1
+|payload |Asset identifier created successfully |yes |link:https://admin-shell-io.github.io/aas-specs-antora/IDTA-01001/v3.1/spec-metamodel/core.html#SpecificAssetId[SpecificAssetId] |1..*
+|===
+
+==== Operation PostAllAssetInformationById
+
+[.table-with-appendix-table]
+[width=100%,cols="25%,35%,10%,20%,10%"]
+|===
+h|Operation Name 4+e|[[PostAllAssetInformationById]]PostAllAssetInformationById
+h|Explanation 4+a|
+Creates or replaces the AssetInformation linked to an Asset Administration Shell to edit discoverable content.
```

Comment:
I think it should be an update information because if access rules do not allow you to read everything you cannot overwrite it.